### PR TITLE
Update Aligner model and tutorial to add NGC checkpoint loading

### PR DESCRIPTION
**Pull request by redoctopus**
Signed-off-by: Jocelyn Huang <jocelynh@nvidia.com>

# What does this PR do ?

Adds NGC model loading for the RAD-TTS Aligner.

**Collection**: TTS
# Changelog 
- Add newly published NGC checkpoint to list of pretrained Aligner models
- Update tutorial to load this checkpoint instead of a local one

# Usage

```python
from nemo.collections.tts.models import AlignerModel

aligner = AlignerModel.from_pretrained("tts_en_radtts_aligner")
```
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [x] Documentation

### CODE REVIEW

**Review by XuesongYang — CHANGES_REQUESTED** [review]
ran your tutorial a couple times, but still hit error like,
```
Error(s):	`pynini` not installed, please install via NeMo/nemo_text_processing/pynini_install.sh
```

See my gist here: https://colab.research.google.com/gist/XuesongYang/918a2ecc7c93cde7c65f3fa0a1241606/aligner_inference_examples.ipynb

File: nemo/collections/tts/models/aligner.py [hunk 1]
```diff
@@ -230,6 +231,19 @@ def setup_test_data(self, cfg):
         pass
 
     @classmethod
-    def list_available_models(cls):
-        """Empty."""
-        pass
+    def list_available_models(cls) -> 'List[PretrainedModelInfo]':
```
Comment (XuesongYang) [inline nemo/collections/tts/models/aligner.py:234]: nit: do we still need a string literal for the type annotation if you've imported the class at Line 27? According to PEP 484, we are good without the quotes (cf https://peps.python.org/pep-0484/#forward-references). Sorry I am not sure about the code convention of NeMo.

**Comment by ekmb** [comment]
> ran your tutorial a couple times, but still hit error like,
> 
> ```
> Error(s):	`pynini` not installed, please install via NeMo/nemo_text_processing/pynini_install.sh
> ```
> 
> See my gist here: https://colab.research.google.com/gist/XuesongYang/918a2ecc7c93cde7c65f3fa0a1241606/aligner_inference_examples.ipynb

@redoctopus, we should add the following to TTS notebooks:
```
! wget https://raw.githubusercontent.com/NVIDIA/NeMo/main/nemo_text_processing/install_pynini.sh
! bash install_pynini.sh
```

File: nemo/collections/tts/models/aligner.py (hunk 1 above)
Comment (redoctopus) [reply nemo/collections/tts/models/aligner.py]: Just took a look through some of our code, and it looks like we have a mix. A lot of List ones are in quotes, a lot of Optional ones aren't. Not sure if we want to standardize at some point, but changed for now

**Comment by redoctopus** [comment]
@ekmb Thanks!

I've updated the installation in this notebook, and I'll make a PR to update it for all other TTS notebooks shortly.

**Review by XuesongYang — APPROVED** [review]
Thank you for the PR! Added my working gist here: https://colab.research.google.com/gist/XuesongYang/43ed17f0942d6bc4f21cbd25f00058ed/aligner_inference_examples.ipynb

File: nemo/collections/tts/models/aligner.py (hunk 1 above)
Comment (XuesongYang) [reply nemo/collections/tts/models/aligner.py]: @ericharper do you have any advice on this? Thanks.

**Pull request merged**